README.md

```diff
@@ -36,20 +36,109 @@
 
 ## Usage
 
-1. Clone this repository locally
-2. Open the app's settings
-3. Select `import theme` and browse to where you cloned Catppuccin
-4. Select it
+1. Import the NixOS and [home-manager](https://github.com/nix-community/home-manager) modules
+
+<details>
+<summary>With Flakes</summary>
```

Comment:
Should we have separate instructions for HM only?

Reply:
you mean for a standalone install?

Reply:
I mean if someone is using Nix and HM on Ubuntu or something and doesn't want the NixOS module.

Reply:
yeah that'd be a [standalone install](https://nix-community.github.io/home-manager/index.html#sec-install-standalone)